apps/jobs/views.py

```diff
@@ -61,13 +63,22 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
         response_data = {'error': 'You haven\'t participated in the challenge'}
         return Response(response_data, status=status.HTTP_403_FORBIDDEN)
 
-    serializer = SubmissionSerializer(data=request.data,
-                                      context={'participant_team': participant_team,
-                                               'challenge_phase': challenge_phase,
-                                               'request': request
-                                               })
-    if serializer.is_valid():
-        serializer.save()
+    if request.method == 'GET':
```

Comment:
`GET` method should be at the start of the view since most of these checks doesn't apply for the `GET` request.